app/assets/stylesheets/calagator/index.scss

```diff
@@ -5,6 +5,7 @@
  * Then come the external dependencies:
  *
  *= require jquery.timepicker
+ *= require jquery-ui/theme
```

Comment:
Oh, that's interesting, I didn't realize vanilla sprockets can't `@import` assets from gems. I've been doing that for years, but use compass on all our projects, so that functionality must come from compass. That explains this current state of affairs, then.

Makes me wonder, though... what if you want to write a library that provides sass mixins? Would the gem consumer be forced into adopting a compass dependency? Seems like a pretty large and obvious hole in sprockets.